manifests/Easier Korath Automatons.yaml

```
@@ -0,0 +1,13 @@
+name: Easier Korath Automatons
+authors: Bereskatuket (Kestrel1110)
+homepage: https://github.com/kestrel1110/Easier-Korath-Automatons
+license: GPL-3.0-or-later
+version: 1.0.0
+shortDescription: Makes capturing Korath Automata easier.
+description: Decreases self-destructing chances by half to make capturing the Korath automata ships easier.
+url: https://github.com/kestrel1110/Easier-Korath-Automatons/archive/refs/tags/1.0.0.zip
+iconUrl: https://raw.githubusercontent.com/kestrel1110/Easier-Korath-Automatons/master/icon.png
```

This should also be versioned, and for style points match the `iconUrl` in `autoupdate`
```suggestion
iconUrl: https://github.com/kestrel1110/Easier-Korath-Automatons/raw/1.0.0/icon.png
```